@xen-orchestra/lite/src/components/pool/network/PoolHostInternalNetworkTable.vue

```diff
@@ -0,0 +1,202 @@
+                  class="text-ellipsis"
+                >
+                  {{ (row.value as XenApiNetwork).name_description }}
+                </div>
+                <div v-if="column.id === 'MTU'" v-tooltip="{ placement: 'bottom-end' }" class="text-ellipsis">
+                  {{ (row.value as XenApiNetwork).MTU }}
+                </div>
+                <div
+                  v-if="column.id === 'default_locking_mode'"
+                  v-tooltip="{ placement: 'bottom-end' }"
+                  class="text-ellipsis"
+                >
+                  {{ (row.value as XenApiNetwork).default_locking_mode }}
+                </div>
+                <div v-if="column.id === 'more'">
+                  <VtsIcon accent="info" :icon="faEllipsis" />
+                </div>
+              </td>
+            </tr>
+          </tbody>
+        </VtsTable>
+      </div>
+      <UiTopBottomTable
+        :selected-items="selected.length"
+        :total-items="usableRefs.length"
+        @toggle-select-all="toggleSelect"
+      />
+    </div>
+  </div>
+  <UiCardSpinner v-if="!isReady" />
+</template>
+
+<script setup lang="ts">
+import UiCardSpinner from '@/components/ui/UiCardSpinner.vue'
+import useMultiSelect from '@/composables/multi-select.composable'
+import type { XenApiNetwork } from '@/libs/xen-api/xen-api.types'
+import VtsIcon from '@core/components/icon/VtsIcon.vue'
+import ColumnTitle from '@core/components/table/ColumnTitle.vue'
+import VtsTable from '@core/components/table/VtsTable.vue'
+import UiButton from '@core/components/ui/button/UiButton.vue'
+import UiButtonIcon from '@core/components/ui/button-icon/UiButtonIcon.vue'
+import UiCheckbox from '@core/components/ui/checkbox/UiCheckbox.vue'
+import UiQuerySearchBar from '@core/components/ui/query-search-bar/UiQuerySearchBar.vue'
+import UiTableActions from '@core/components/ui/table-actions/UiTableActions.vue'
+import UiTitle from '@core/components/ui/title/UiTitle.vue'
+import UiTopBottomTable from '@core/components/ui/top-bottom-table/UiTopBottomTable.vue'
+import { useTable } from '@core/composables/table.composable'
+import { vTooltip } from '@core/directives/tooltip.directive'
+import type { IconDefinition } from '@fortawesome/fontawesome-common-types'
+import {
+  faAlignLeft,
+  faCaretDown,
+  faEdit,
+  faEllipsis,
+  faHashtag,
+  faPlus,
+  faTrash,
+} from '@fortawesome/free-solid-svg-icons'
+import { computed, ref, watchEffect } from 'vue'
+
+const props = defineProps<{
+  hostInternalNetwork: XenApiNetwork[]
+  isReady: boolean
+}>()
+
+const reactiveHostInternalNetworks = ref<XenApiNetwork[]>(props.hostInternalNetwork || [])
+
+const searchQuery = ref('')
+
+const filteredNetworks = computed(() => {
+  return searchQuery.value
+    ? reactiveHostInternalNetworks.value.filter(network =>
+        Object.values(network).some(value => String(value).toLowerCase().includes(searchQuery.value.toLowerCase()))
+      )
+    : reactiveHostInternalNetworks.value
+})
```

Comment:
I'd suggest something like that for better readability:

```suggestion
const filteredNetworks = computed(() => {
  const searchTerm = searchQuery.value.trim().toLocaleLowerCase()

  if (!searchTerm) {
    return networks.value
  }

  return networks.value.filter(network =>
    Object.values(network).some(value => String(value).toLocaleLowerCase().includes(searchTerm))
  )
})```